packages/concerto-core/test/composer/composermodel.js

```diff
@@ -14,8 +14,7 @@
 
 'use strict';
 
-const SYSTEM_MODELS = [
-
+const COMPOSER_MODEL =
```

Comment:
Can we remove this file entirely?

Reply:
Ah, this file is only used as a utility to make the tests work unchanged, right?